mdpow/fep.py

```diff
@@ -693,7 +694,9 @@ def dgdl_edr(self, *args):
         if not os.path.exists(fn):
             logger.error("Missing dgdl.edr file %(fn)r.", vars())
             raise IOError(errno.ENOENT, "Missing dgdl.edr file", fn)
-        return fn
+        edrs = glob.glob(*args + (self.deffnm + '*.edr',))
```

Comment:
This looks as if it might wrongly raise an exception if there isn't a deffnm.edr file present.

How about something like (removing lines start ing at `fn = ...`)
```python
pattern = os.path.join(args) + self.deffnm + '*.edr'
edrs = glob.glob(pattern)
if not edrs:
            logger.error("Missing dgdl.edr file %(pattern)r.", vars())
            raise IOError(errno.ENOENT, "Missing dgdl.edr file", pattern)
return [os.path.abspath(i) for i in edrs]
```

(Btw, I would be surprised if `edrs = glob.glob(*args + (self.deffnm + '*.edr',))` actually worked.)